wsmaster/integration-tests/mysql-tck/mysql-support/src/main/resources/che-schema111/5.0.0-M9/mysql/2__update_local_links_in_environments.sql

```diff
@@ -0,0 +1,14 @@
+--
```

Comment:
What is the purpose of `che-schema111`  set ?

Reply:
@mshaposhnik Thanks for catching this.  I meant to remove it.